### ISSUE TITLE
Warnings in SDK

### ISSUE THREAD

**Issue by paulbruneau** [Iterable/swift-sdk#420]
Hello-
For a few versions now, there have been these warnings in the Iterable SDK. If you'd like I can attempt to fix them and create a PR for you, but I thought I would raise the issue here first in case it's already being addressed in an upcoming version. Thank you.

<img width="1194" alt="Screen Shot 2020-10-30 at 10 07 20 AM" src="https://user-images.githubusercontent.com/978400/97714894-1c796100-1a98-11eb-8945-6384660108a3.png">
<img width="1201" alt="Screen Shot 2020-10-30 at 10 08 03 AM" src="https://user-images.githubusercontent.com/978400/97714897-1d11f780-1a98-11eb-8d65-767933565935.png">

**Comment by paulbruneau** [comment]
Thanks @roninopf ! I agree it is something going on with the objc-swift mechanism of Xcode. I will take a shot at it because I'm in the middle of translating a bunch of objc to swift right now so the topic is of interest to me.

**Comment by paulbruneau** [comment]
Hi @roninopf --

I haven't been able to figure out a way to remove the warning while keeping the \param. My only suggestion is to remove the \param docs. Maybe they can be still referenced, just not as \param because it does seem like xcode is reading or creating them incorrectly on the objc side. Thanks for looking at this with me.

**Comment by roninopf** [comment]
Thanks @paulbruneau, we'll put it up on the list, looks easy enough to deal with. Another alternative we're considering is also to rename those fields to match their Objective-C counterparts rather than the Swift names.

**Comment by paulbruneau** [comment]
Thank you @roninopf that sounds great. I didn't want to suggest that because who wants to go backwards, right? But during these "crossover times" we have to do some interesting things sometimes. Cheers!

**Comment by roninopf** [comment]
Hi @paulbruneau, just writing in to say we haven't forgotten about this, we're just looking for a good time to get it done.

**Comment by paulbruneau** [comment]
Thanks! I understand completely.

But hey, thanks to github's email they sent me about this, I see your name might be Jay Kim. Have you ever heard of anyone else seeing this xcode debug console error when trying to "PO" values? I experienced it quite a bit the other day.

`error: virtual filesystem overlay file ‘/Users/jay.kim/Library/Developer/Xcode/DerivedData/swift-sdk-dcpbsnbeyjmqjqahjaffacrgnwlg/Build/Intermediates.noindex/ArchiveIntermediates/swift-sdk/IntermediateBuildFilesPath/swift-sdk.build/Release-iphonesimulator/swift-sdk.build/all-product-headers.yaml’ not found`

**Comment by roninopf** [comment]
Yep, that's me! Wow, that's very odd. This is the first report of this we've seen. We'll make a ticket for this, thanks.

**Comment by paulbruneau** [comment]
Thanks! It could be something weird with Xcode generally, or just my own Xcode or project situation. I have no idea what a "virtual filesystem overlay file" is especially regarding the simulator. I'll let you know if I get any more interesting information about it from my team.

**Comment by paulbruneau** [comment]
Hi again @roninopf -- 
Just reporting back that another member of my team is experiencing the debug console problem when trying to "PO" values in there. Should I make a new issue for this? Have you heard from anyone else about this one? Thanks!

```
(lldb) po ratesDict
error: virtual filesystem overlay file '/Users/jay.kim/Library/Developer/Xcode/DerivedData/swift-sdk-dcpbsnbeyjmqjqahjaffacrgnwlg/Build/Intermediates.noindex/ArchiveIntermediates/swift-sdk/IntermediateBuildFilesPath/swift-sdk.build/Release-iphonesimulator/swift-sdk.build/all-product-headers.yaml' not found
error: couldn't IRGen expression. Please check the above error messages for possible root causes.
```

**Comment by roninopf** [comment]
Hmm, no other reports, but can you make a ticket with your Iterable CSM? I'm thinking this will take a little more digging than can be encompassed in a GitHub issue. :/

**Comment by roninopf** [comment]
Hi @paulbruneau, this is a long time coming, but this issue has finally been resolved with 6.3.3. Thanks for reporting this.
